apps/system/js/dialer_agent.js

```diff
@@ -254,5 +258,12 @@
     }
   };
 
+  DialerAgent.prototype.onCall = function() {
```

Comment:
we have 2 `numOpenLines > 0` checks in `handleEvent` that should be replaced by this